LayoutTests/imported/w3c/web-platform-tests/shadow-dom/declarative/declarative-shadow-dom-basic.tentative-expected.txt

```diff
@@ -1,6 +1,7 @@
 
 PASS Declarative Shadow DOM: Basic test
 FAIL Declarative Shadow DOM: Feature detection assert_true: Unable to feature detect expected true got false
+FAIL Shadowrootmode reflection assert_equals: The shadowRootMode IDL should reflect the content attribute expected (string) "open" but got (undefined) undefined
```

Comment:
These 2 failures here don't seem great.

Reply:
Yeah, these are new test cases that are failing. Will post a patch to fix it as a followup.